tests/chaos/__init__.py

```
@@ -0,0 +1,311 @@
+    mock_response = AsyncMock()
+    mock_response.status = 200
+    mock_response.json = AsyncMock(return_value={
+        "system": {"status": "HEALTHY"}
+    })
+    
+    chaos_engine.session.get = AsyncMock(return_value=mock_response)
+    chaos_engine.session.get.return_value.__aenter__ = AsyncMock(return_value=mock_response)
+    chaos_engine.session.get.return_value.__aexit__ = AsyncMock(return_value=None)
+
+    # Test that injection counter increments
+    initial_count = CHAOS_INJECTIONS.labels(fault_type="model_loader")._value.get() if hasattr(CHAOS_INJECTIONS, '_value') else 0
```

Ruff (0.14.10)

Local variable `initial_count` is assigned to but never used

Remove assignment to unused variable `initial_count`

(F841)